gcp-vertex-ai-mojo-scorer/src/main/java/ai/h2o/mojos/deploy/gcp/vertex/ai/config/EnvironmentConfiguration.java

```diff
@@ -66,7 +73,7 @@ private Properties parseGcsPath(String gcsPath) {
 
   private String getFromEnv(Map<String, String> env, String envVar) {
     String envValue = env.getOrDefault(envVar, "");
-    if (envValue.isEmpty()) {
+    if (envValue.isEmpty() && !envVar.equals("PREPROCESSING_SCRIPT_PATH")) {
```

Comment:
nit: this seems like some helper method, but there is now a side effect of checking if said arg is `PREPROCESSING_SCRIPT_PATH`, it is better to avoid this entirely I would think.